ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -105,11 +109,8 @@ Verify Pipeline Metadata Pods Are Not Deployed When Running Pipelines
 Pipelines Suite Setup    # robocop: disable
     [Documentation]    Sets global test variables, create a DS project and a data connection
     Set Library Search Order    SeleniumLibrary
+    # TODO: Install Pipeline only if it does not already installed
     Install Red Hat OpenShift Pipelines
-    ${prj_title}=    Set Variable    ${PRJ_BASE_TITLE}-${TEST_USER_3.USERNAME}
-    ${iris_pipeline_name}=    Set Variable    ${PIPELINE_TEST_BASENAME}-${TEST_USER_3.USERNAME}
-    Set Suite Variable    ${PRJ_TITLE}    ${prj_title}
-    Set Suite Variable    ${PIPELINE_TEST_NAME}    ${iris_pipeline_name}
```

Comment:
why removing these lines?

Reply:
Since these are widely used variables, I moved them under *** Variables *** section:
```
${PRJ_BASE_TITLE}=   dsp
${PRJ_DESCRIPTION}=   ${PRJ_BASE_TITLE} is a test project for validating DS Pipelines feature
```
https://github.com/red-hat-data-services/ods-ci/pull/995/files#diff-0e3e8f6733ca76331b5ae98a47739e91267cb03bd98a02b4d57615fe8ea4480aR16-R17

Reply:
ah, okay :)